setup.py

```diff
@@ -1,9 +1,12 @@
 from setuptools import setup, find_packages
 
 
-version = '0.1.2'
+version = '0.1.3'
 
-requires = []
+requires = [
+    'pysaml2>=1.0.0',
+    'redis>=2.10.5',
```

Comment:
eduid_common is a namespace package, so at any moment we can put eduid_common.session (and/or eduid_common.authn) into its own github repo/pypi egg, removing the dependencies from core eduid_common.